src/datomish/datom.cljc

```diff
@@ -0,0 +1,137 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+;; Purloined from DataScript.
```

Comment:
I don't know, and neither did @Mossop.  I'll email @tonsky about a license grant, or we can rewrite this.